lib/sitemap_generator.js

```diff
@@ -48,12 +51,28 @@ module.exports = {
 		var urlset_xml = [];
 
 		for (var i = 0; i < self.urls.length; i++) {
-			urlset_xml.push( "<url>" +
-											 "<loc>" + self.urls[i].loc + "</loc>" +
-											 "<lastmod>" + self.urls[i].lastmod + "</lastmod>" +
-											 "<changefreq>" + self.urls[i].changefreq + "</changefreq>" +
-											 "<priority>" + self.urls[i].priority + "</priority>" +
-											 "</url>" );
+			var changefreq;
+			var priority;
+
+			if (self.urls[i].changefreq === "false") {
```

Comment:
I wouldn't even put this logic in here I would just drop these elements from the xml.

Reply:
oh ok